### PR TITLE
[7.2.x] JBPM-6027: Advanced Search API shows error code 500 instead of 400

### PR DESCRIPTION
**Pull request by markcoble**
@domhanak one more attempt.  I did not test against the various DBs but  DataSetLookupException is what is thrown when the BAD REQUESTs are encountered.  Please test and let me know.

### CODE REVIEW

**Comment by domhanak** [comment]
@markcoble Sure thanks, I'll give it a go. Does the fail seem related or is there a problem with CI right now?

**Comment by domhanak** [comment]
@markcoble this is fine with me if checks pass, could you please create prq for master too?

**Comment by domhanak** [comment]
jenkins retest this

File: kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/pom.xml [hunk 1]
```diff
@@ -101,6 +101,11 @@
     </dependency>
 
     <dependency>
+      <groupId>org.dashbuilder</groupId>
```
Comment (domhanak) [inline kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/pom.xml]: I believe this is breaking the build? @markcoble Could you please check?

Comment (markcoble) [inline kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/pom.xml]: just pushed the correct dependency

**Comment by domhanak** [comment]
@markcoble could you prefix this prq with [7.2.x] and create one for master aswell? :)

**Pull request merged**